notebook-controller/tox.ini

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+[tox]
+skipsdist = True
+skip_missing_interpreters = True
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}
+    PYTHONBREAKPOINT=ipdb.set_trace
+    CHARM_REPO=https://github.com/canonical/notebook-operators.git
+    CHARM_BRANCH=main
+    LOCAL_CHARM_DIR=charm_repo
+
+[testenv:unit]
```

Comment:
The test in notebook-controller/tests/ doesn't seem like a unit test, maybe we want to rename this testenv to something more appropriate.

Reply:
Yes, it is like a smoke or integrtiry test. I thought, I'd keep same environment names, but may be for ROCKs we can rename it into something different.

Reply:
To keep it consistent with our other workflows and environments it makes sense to keep it as testenv:unit.
In reality it is a unit test for the image, i.e. testing images bits and pieaces. Keeping it consistent will allow use to use common CI for ROCKs in the future.